client/src/components/EditProfile/ProfileForm/ProfileForm.tsx

```diff
@@ -0,0 +1,197 @@
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { Formik } from 'formik';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { CircularProgress } from '@material-ui/core';
+import DatePicker from '../../DatePicker/DatePicker';
+
+const genders = [
+  { value: 'Male', label: 'Male' },
+  { value: 'Female', label: 'Female' },
+  { value: 'Other', label: 'Other' },
+];
+
+export default function EditProfile(): JSX.Element {
+  const classes = useStyles();
+
+  const handleSubmit = () => {
+    return;
+  };
+
+  return (
+    <Formik
+      initialValues={{
+        firstName: '',
+        lastName: '',
+        gender: '',
+        email: '',
+        address: '',
+        description: '',
+      }}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <Grid container>
+          <form onSubmit={handleSubmit} className={classes.form} noValidate>
+            <Grid item className={classes.field}>
+              <Grid className={classes.label}>
```

Comment:
```suggestion
              <Grid item className={classes.label}>
```
We always need to specify the type (item/container/item container) of the Grid item. Ditto throughout!